Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -77,14 +75,16 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
   }
 
   private func setRewardTitleArea(project: Project) {
-    if project.personalization.isBacking != true && project.state == .live {
+    if project.personalization.isBacking != true, project.state == .live {
```

Comment:
Ah interesting.. do we prefer the comma to double-ampersand for this?

Reply:
I think we do.

Reply:
Yeah, I can revert this back to `&&`, `||`, etc.

Reply:
I think comma is fine, it's more consistent with how we'd express a `guard` or `if let`